compiler/src/iree/compiler/Codegen/LLVMCPU/LLVMCPUTileAndFuse.cpp

```diff
@@ -145,6 +154,12 @@ LogicalResult applyTileAndFuse(RewriterBase &rewriter, Operation *rootOp,
     // Traverse the slices in BFS fashion.
     tensor::ExtractSliceOp candidateSliceOp = candidates.front();
     candidates.pop_front();
+    if (mapToIterArg.contains(candidateSliceOp.getSource())) {
```

Comment:
So you shouldnt be doing this for all `tensor.extract_slices`. You should only do this for `tensor.extract_slice` whose results are used for the destination operands of operations.